tests/client.test.js

```diff
@@ -37,7 +38,8 @@ tap.test('Flow Manager Client', async t => {
   } = await kafkaCommon.initializeKafkaClient(conf)
 
   const kafkaConf = { clientId: conf.KAFKA_CLIENT_ID, brokers: conf.KAFKA_BROKERS_LIST }
-  const consumerConf = { groupId: conf.KAFKA_GROUP_ID }
+  const consumerGroup = `${conf.KAFKA_GROUP_ID}_${crypto.randomBytes(12).toString('hex')}`
```

Comment:
same here why random?

Reply:
@fredmaggiowski as @SimoneBergonzi said, I added the random part to run tests in parallel without problems.